kfs-core/src/main/resources/edu/arizona/kfs/coa/identity/ObjectSubTypeCodeRoleTypeServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2009 The Kuali Foundation.
+ * 
+ * Licensed under the Educational Community License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.opensource.org/licenses/ecl1.php
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.arizona.kfs.coa.identity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.rice.kew.doctype.service.DocumentTypeService;
+import org.kuali.rice.kew.service.KEWServiceLocator;
+import org.kuali.rice.kim.bo.types.dto.AttributeDefinitionMap;
+import org.kuali.rice.kim.bo.types.dto.AttributeSet;
+import org.kuali.rice.kim.service.support.impl.KimRoleTypeServiceBase;
+import org.kuali.rice.kim.util.KimCommonUtils;
+import org.kuali.rice.kns.datadictionary.AttributeDefinition;
+
+import edu.arizona.kfs.sys.identity.ArizonaKfsKimAttributes;
+
+public class ObjectSubTypeCodeRoleTypeServiceImpl extends KimRoleTypeServiceBase {
+    DocumentTypeService documentTypeService;
+    
+    @Override
+    protected boolean performMatch(AttributeSet qualification, AttributeSet roleQualifier) {    
```

Comment:
Lets clean up this method code and only reference the edu.arizona version of KfsKimAttributes.